invenio_app_ils/ui/src/invenio_app_ils/common/config.js

```diff
@@ -9,3 +9,4 @@ export const invenioConfig = getInvenioConfig();
 // NOTE: the delay that is used when we fire a request with setTimeOut
 // in order to give some time to elastic search to index.
 export const ES_DELAY = 3000;
+export const SUPPORT_EMAIL = 'cds-support@cern.ch';
```

Comment:
Here I think what @ntarocco was saying is to take this value from `invenio-config`, am I right?

Reply:
@ntarocco which config you were referring too? I think @zzacharo is right, it should probably be the `invenio-config`